.changeset/ssr-manifest-breaking.md

```diff
@@ -0,0 +1,12 @@
+---
+'astro': major
+---
+
+**SSRManifest structure changes:** Path properties now strictly require `URL` objects, and several new required properties have been added.
```

Comment:
Do we need to document this change? Feels somewhat internal

Reply:
Adapters receive this object, so technically public even though they likely just pass to the App class.